nlp/2021_06_18_data_augmentation/totw_nlp_dat_aug.ipynb

```diff
@@ -0,0 +1,6046 @@
+          "height": 360,
+          "referenced_widgets": [
+            "909a3e24f580400986794f63ce2d0259",
+            "3be76b41c86b4f6da040452cd7aa8405",
+            "b7d60f19972440f28e2f4b3fbc6d8b54",
+            "97091bc2ad9f4465abe8c06d8c116a93",
+            "96baf48472c54cffa057daa78abfe5a7",
+            "5f2f0c8381334de9bd6b01d1c5bfe7ee",
+            "42345e0475f94f0eaa722883fcc19b0e",
+            "7d50cbfb19494372bd62e3345ca3d2e5",
+            "c036a14f63b64df6acfdd70963aa8fff",
+            "fee2b0d3ea724c0fb987e69ce79ca386",
+            "611e926d270747d59f2b5e1c057f4d66",
+            "8b90ea4422c14e3db9f5ff4188bc0f49",
+            "874e9c2acbe64bcda5631051367eeaa6",
+            "ffa10472dc0340ec9dad55f9983ec25b",
+            "84e051427fe64782afc52512224ff32a",
+            "54f1cb732e344838b2f5ab7fae2f2e66",
+            "767229e1298345d3bf7fc04b56323f4d",
+            "3bb186f31a2b46509d5b5b37538836e0",
+            "d183eb249b224839b2010be889297d3e",
+            "42690ac297da4e2fa03db6a3c423d271",
+            "a69346bea1714af2ae2e93991f61c00b",
+            "78f687135d55492b950e1fc9a1e49fc9",
+            "7c08a7470f6b49928446e84c88c00fc9",
+            "d9c45eae855245f79bdda62e2659a9cd",
+            "4b243b1e85dd4a55b0dc1a770eb8d6c3",
+            "df00f8a9b18d41de8036cfcbfa0795cb",
+            "e786e1bbcb644508869e75761773a0cf",
+            "d285ee2cbccd493b9a59ef75e86e0dcc",
+            "b6acc51b861844a790712bd295645a45",
+            "f683b0b2b53043a8afd164386925a3f7",
+            "4abc5e990d6249dcb5c5fc59728b758b",
+            "1e2de85a802c4686b4deb2c0fa008838",
+            "020b840abade41df8c006a9871ba7daa",
+            "5ef448f79dcd47119c59eb5dbf7520d4",
+            "f4fa3d4a19f9468da56d9b7e33e03903",
+            "527ee0ce2fd1450abeb0557346fdfb23",
+            "d25693d4e92f49529aea775a00c1fa76",
+            "1153984fd81e44278881f0e8ba9a38f1",
+            "a682f16511034230893b5bea4372671d",
+            "f42f35ee386448e4a6aa1515c47b8912",
+            "e807085786914738924f238cd07592d6",
+            "dd61a03b3a644853bb2ee5aa44c0a887",
+            "afd3f0eb672b42568c49c39badea4fed",
+            "56422ef80b5d449f9b830070f280a276",
+            "77f9b1b8f36a4bb3b94e14f1acddaad8",
+            "392ce1ca68b54d99ba77784ff43d1443",
+            "966f5c34164d49c6a9ce1f30a59857e6",
+            "c711e34d31fe4b0f80ac204583a85b0e",
+            "8e447e52a88045c689ea9c6b7edcc622",
+            "1d76a8ad5d6148398bba612f57ebfdeb",
+            "1beab6dfbd7c4076bfa6835fcdf42862",
+            "ed015b890159444988fb18aaa054a275",
+            "8fed43a8bbbb4d7cb02cc5440d87b481",
+            "8ccd26e54cbb4f5193f007066b1c9606",
+            "99d38d6df4394093b04bd57354ddbcb2",
+            "6a963434eca74d18a4c9a1952f5c9b4d"
+          ]
+        },
+        "outputId": "5d989989-8f82-433c-91e1-62d72b561d78"
+      },
+      "source": [
+        "def back_tranlation_nl_fr_en_nl(texts):\n",
```

Comment:
I would define this method in the section on backtranslation, it's a bit confusing that it's here